nionswift_plugin/usim/CameraDevice.py

```diff
@@ -151,7 +151,7 @@ def acquire_image(self) -> dict:
         xdata_buffer = None
         integration_count = self.__integration_count or 1
         for frame_number in range(integration_count):
-            if not self.__has_data_event.wait(self.__exposure * 200):
+            if not self.__has_data_event.wait(self.__exposure * 200) and not self.__thread.is_alive():
```

Comment:
Can you explain how this situation of timeout with thread alive arises?

Reply:
I had the issue when using the MultiAcquire code with the simulator. There, multiple calls to "acquire_sequence" are executed with almost no time in between. While processing/transferring the data the simulator thread sometimes get stuck for a short while which will trigger the exception here. With the additional check this does not happen anymore. And if there is really an exception in the simulator thread that causes it to die, we will get the exception here, too.